### PR TITLE
Sanlouise 14881 add cypress tests profile

### PR DESCRIPTION
**Pull request by sanlouise**
## Description
This PR replaces the following 2 test files:

1. https://github.com/department-of-veterans-affairs/vets-website/blob/d1f7243257c92e37acb0b1dabfaa10dceb29519d/src/platform/user/profile/vet360/tests/e2e/addressValidation.e2e.spec.js
2. https://github.com/department-of-veterans-affairs/vets-website/blob/d1f7243257c92e37acb0b1dabfaa10dceb29519d/src/applications/personalization/profile360/tests/e2e/profile-360.e2e.spec.js

I left out editing the email address from the profile test because it should be a separate test if it hasn't already been covered.

This PR merely replaces deleted functionality and is not intended to provide better coverage for address validation. [This ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14987) will be responsible for adding good e2e coverage for address validation and verifying that we have enough coverage for updating the email address.

## Testing done
Tests pass locally

## Acceptance criteria
- [x] Create cypress tests

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-address-validation.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }).click({
+      force: true,
+    });
+
+    // Update address
+    cy.get(`#root_addressLine3`)
+      .click()
+      .type('Addition to Address');
+
+    cy.findAllByText(/Update/)
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-address-validation.cypress.spec.js:45]: This seems fragile. Does `findByRole('button', {name: 'Update'})` work?

File: src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-fields-check.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    // Check service branch
+    cy.findByText(/United States Air Force/i).should('exist');
+
+    // Check date of birth
+    cy.findByText(/May 6, 1986/i).should('exist');
+
+    // Check gender
+    cy.findByText(/Male/i).should('exist');
+
+    // Check home address
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-fields-check.cypress.spec.js:44]: nit: check mailing address

File: src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-address-validation.cypress.spec.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      .click();
+
+    // Ensure correct addresses show up
+    cy.findByText(
+      /1493 Martin Luther King Rd, Apt 1, Addition to Address/i,
+    ).should('exist');
+
+    cy.findByText(/400 NW 65th St/i).should('exist');
+
+    cy.findAllByText(/Update/)
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-address-validation.cypress.spec.js:56]: Same here.

**Review by erikphansen — APPROVED** [review]
Just a couple small comments. Otherwise looks good as replacements for the old Nightwatch tests.

File: src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-address-validation.cypress.spec.js (hunk 1 above)
Comment (sanlouise) [reply src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-address-validation.cypress.spec.js]: Updated! 👍

File: src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-fields-check.cypress.spec.js (hunk 2 above)
Comment (sanlouise) [reply src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-fields-check.cypress.spec.js]: Good catch!

**Pull request merged**